INSTALL.md

```diff
@@ -55,6 +55,24 @@ ruby -v
 
 > Mac OS X 10.13 comes with 2.3.7 but you might also use [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install your own ruby version.
 
+#### Node version
+
+The project supports **LTS Version: 10.13.0 (npm 6.4.1)**.
+
+You might want to use [nvm](https://github.com/creationix/nvm/) to install and work with a specific Node version:
+
+```bash
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+```
+
+Restart the terminal and install Node:
+
+```bash
+nvm install 10.13.0
+nvm use 10.13.0
```

Comment:
Is there a difference between Node 10.x versions?

Reply:
well, minor TBH, we try to stick with latest LTS

Reply:
so, `brew install node@10` should be good too no?

Reply:
Yes, I could add as a macos alternative